app/main.py

```diff
@@ -1 +1,40 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
The `feed` method in the `Animal` class could be simplified. According to the checklist, it's better to avoid using an extra `else` statement. You can simply return 0 after the `if` statement. This is because if `self.is_hungry` is `True`, the function will return in the `if` statement and never reach the line after it. If `self.is_hungry` is `False`, the `if` statement will be skipped and the function will return 0. This makes the code cleaner and easier to read. Here is how you can improve it: python def feed(self) -> int: if self.is_hungry: print(f"Eating {self.appetite} food points...") self.is_hungry = False return self.appetite return 0 ```